utilities/common.py

```diff
@@ -232,7 +232,15 @@ def __call__(self, parser, namespace, values, option_string=None):
             # since the dataAltSig tag-and-probe fits were not run in 3D (it is assumed for simplicity that the syst/nomi ratio is independent from uT)
             #
             # 2D SF without ut-dependence, still needed to compute systematics when uing 3D SF
-            sfFile = "allSmooth_GtoHout.root" if commonargs.isolationDefinition == "iso04" else "allSmooth_GtoHout_vtxAgnIso.root"
+            if commonargs.era == "2016PostVFP":
+                sfFile = "allSmooth_GtoHout.root" if commonargs.isolationDefinition == "iso04" else "allSmooth_GtoHout_vtxAgnIso.root"
+            elif commonargs.era == "2018":
+                if commonargs.isolationDefinition == "iso04":
+                    raise NotImplementedError(f"For Era {commonargs.era} Isolation Definition {commonargs.isolationDefinition} is not supported")
+                else:
+                    sfFile = "allSmooth_2018_vtxAgnIso.root"
+            else:
+                raise NotImplementedError(f"Era {commonargs.era} is not yet supposrted")
```

Comment:
supposrted --> supported, anyway I think the check on the era happens in multiple places already , so we may not need to raise when the passed era is not valid

Reply:
This looks to me like filler for an unimplemented 2017 case. It's fine as is for now, I think.